### DISCUSSION TITLE
Smart Search indexer not updating when indexed content updates.

### DISCUSSION THREAD

**Discussion by Chaosxmk** [joomla/joomla-cms#39377] [Q&A]
I'm trying to setup a smart search plugin. I've successfully gotten it up and running to a basic degree where Smart search properly indexes my content and display during a search.

The issue I'm having is that if I update my content, the indexed content doesn't seem to be receiving an update. I tested with articles/categories just to be sure that if a title was changed, the indexed items would reflect that, and they do.

I couldn't find any documentation for this component that I could understand effectively, so I based everything on what I could see `/plugins/finder/content/` doing. I know I'm missing something, but I haven't a clue as to what that would be. Any tips or information would be appreciated.

**Comment by Hackwar** [comment]
Hello @Chaosxmk, your component needs to implement events upon saving and when you are not using the standard events on which the finder content plugin reacts, your plugin needs to be loaded at the right time and trigger the right events. These events have to be in your plugin then. Sorry for all of this to be pretty unspecific, but to help you further, we would need to know more about your code.
